docs/API/Elements.md

```diff
@@ -5,6 +5,24 @@ title: Elements
 
 The Elements API is a powerful abstraction that allows users to manipulate any property on any element through a handful of simple methods. This is important because these few methods allow for easy networking or persistent state. 
 
+When it comes to Javascript Element objects each element has a few public properties. These can be used to reference the object.
+
+```js
+var a = this.id;   // Read-only. Gets the element's ID
+
+var a = this.type; //Read-only. Get's element type.
+```
+The most common form to reference would be:
+
+```js
+var b = this.self; 
```

Comment:
I don't think `this.self` is a valid reference.  If I do `log.info(this.self);` it returns `null`.  Just using `this` on its own should be enough.

`this` in JavaScript is sometimes hard to understand because it is relative.  So `this` might refer to one thing if used in one place, but might refer to something entirely different if used in another place even if both references are in the same script.  For this reason, if you've ever seen us using `var self = this;`, this is intended to cast `this` to a variable when it is referencing the specific context we want.  Because most of the time we don't want `this` to mean something different every time, we want it to mean the same thing every time.

Alternatively, if `this.self` is actually a valid reference that I've not seen before, I think its description needs more context because right now I'm not sure how to use it correctly 😅